cmd/buildah/common.go

```diff
@@ -88,6 +90,11 @@ func getStore(c *cobra.Command) (storage.Store, error) {
 		options.GIDMap = gidmap
 	}
 
+	oldUmask := syscall.Umask(0)
+	if oldUmask != 0 {
+		logrus.Warningf("umask value different than 0.  Forcing it to 0")
```

Comment:
Doesn't this mean that every running of buildah will cause this to happen?  IE Lots and lots of Warning messages.

Reply:
When I set my umask to 0, and create a file, I end up creating a file 0666, which is not what we want, it is a security risk.
Should we set the default to 0002, or 0022

[dwalsh@localhost libpod]$ umask 0000; rm -f ~/walsh; touch ~/walsh; ls -l ~/walsh
-rw-rw-rw-. 1 dwalsh dwalsh 0 Feb  6 05:41 /home/dwalsh/walsh
[dwalsh@localhost libpod]$ umask 0022; rm -f ~/walsh; touch ~/walsh; ls -l ~/walsh
-rw-r--r--. 1 dwalsh dwalsh 0 Feb  6 05:41 /home/dwalsh/walsh
[dwalsh@localhost libpod]$ umask 0002; rm -f ~/walsh; touch ~/walsh; ls -l ~/walsh
-rw-rw-r--. 1 dwalsh dwalsh 0 Feb  6 05:41 /home/dwalsh/walsh

Reply:
Looks like the default login is supposed to be 0022. Although when I setup a new account, I see it as 0002.

Reply:
I can change it to Debug

Reply:
There's this check in /etc/bashrc, it might be related.
```
    if [ $UID -gt 199 ] && [ "`id -gn`" = "`id -un`" ]; then
       umask 002
    else
       umask 022
    fi
```

Reply:
I guess someone decided if you have the same Username and Group name by default then files you create should be 
0664
If not they should be
0644

So wouldn't everything work correctly if the umask was 0022

Reply:
There's also the `$UID -gt 199` condition, so **only** system-defined UIDs like `postgres` or `mariadb` have 0022 umask.  I'm not sure, unless there's a reason to be more restrictive (some playing with volumes regarding group permissions, e.g.) I'd probably go with 0002.